uvvid/cli.py

```diff
@@ -0,0 +1,43 @@
+import click
+import cv2 as cv
+
+from .core import UVVID
+
+DEBUG = False
+
+
+@click.group()
+@click.option('--debug/--no-debug', default=True)
```

Comment:
@Foxiy @Adman note that debug is turned on by default. We might want to change that in the future.